packages/paste-website/src/pages/icons/usage-guidelines/index.mdx

```diff
@@ -43,9 +43,9 @@ export const pageQuery = graphql`
 
 <Callout>
   <CalloutText>
-      Our icon set is still limited, and more will be added over time. Please <Anchor href="https://github.com/twilio-labs/paste/issues">
-      file an icon request
-    </Anchor> or follow the guide on <SiteLink to="/icons/how-to-add-an-icon">how to add an icon</SiteLink>.
+    Our icon set is still limited, and more will be added over time. Please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/issues">file an icon request</Anchor> or follow the guide on{' '}
```

Comment:
Should this link to Discussions now instead of Issues?

Reply:
👍 I feel like thats where we get the requests.

Reply:
Considering we have quite a lot of icons now, should this change to "can't find the icon you need? Check streamline and contribute it to the library following our guidelines."